### PR TITLE
update website release ticker via cms

### PR DESCRIPTION
**Pull request by erquhart**

### CODE REVIEW

**Comment by verythorough** [comment]
Deploy preview for *netlify-cms-www* ready!

Built with commit d785aaa0d751313e1c8d104ae5eb5c783da44695

https://deploy-preview-1144--netlify-cms-www.netlify.com

**Comment by verythorough** [comment]
Deploy preview for *cms-demo* ready!

Built with commit d785aaa0d751313e1c8d104ae5eb5c783da44695

https://deploy-preview-1144--cms-demo.netlify.com

File: website/site/layouts/index.html [hunk 1]
```diff
@@ -36,7 +36,7 @@ <h3>{{ .feature | markdownify }}</h3>
       <ol>
         {{ range .Site.Data.updates }}
           {{ range first 3 . }}
-            <a href="{{ .url }}"><li><div class="update-metadata"><span class="update-version">{{ .version }}</span><span class="update-date">{{ dateFormat "January 2, 2006" .date }}</span></div><span class="update-description">{{ .description  | markdownify }}</span></li></a>
```
Comment (tech4him1) [inline website/site/layouts/index.html]: Why not just use the `url` from the updates.yml?

Comment (erquhart) [reply website/site/layouts/index.html]: That's what this is - `Site.Data.updates` is read in from the yaml.

Comment (tech4him1) [reply website/site/layouts/index.html]: It was changed from `{{ .url }}` to `https://github.com/netlify/netlify-cms/releases/tag/{{ .version }}`. We still have the URL in the `Site.Data.updates`, though, why not just use that url?

Comment (erquhart) [reply website/site/layouts/index.html]: Oh, sorry, should have looked closer.

We're going to drop the urls entirely - that way we don't have to manually enter a url every time, just derive it using the version. Not sure why I didn't drop them.

**Comment by erquhart** [comment]
Agreed.

**Comment by erquhart** [comment]
Merge after #1143, otherwise date formats will be wrong.

**Pull request merged**